model_analyzer/result/model_config_measurement.py

```diff
@@ -67,6 +68,8 @@ def from_dict(cls, model_config_measurement_dict):
         model_config_measurement._non_gpu_data_from_tag = cls._get_non_gpu_data_from_tag(
             model_config_measurement)
 
+        model_config_measurement._metric_weights = {"perf_throughput": 1}
```

Comment:
Won't real values be in the dict?

Reply:
I've updated the code to restore these from the checkpoint and updated the L0's to use the new checkpoint that contains the default values.